### PR TITLE
Fix benchmark queries

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

**Comment by findepi** [comment]
As a follow-up we should remove duplication between product tests and presto-benchto-benchmarks. We should have exactly one copy of the TPC-DS queries to avoid these kind of problems for the rest of our lives.

File: presto-product-tests/src/main/resources/sql-tests/testcases/tpcds/q33.sql [hunk 1]
```diff
@@ -14,7 +14,7 @@ WITH
       SELECT "i_manufact_id"
       FROM
         item
-      WHERE ("i_category" IN ('Electronics'))
+      WHERE ("i_category" IN ('Electronics                                       '))
```
Comment (kokosing) [inline presto-product-tests/src/main/resources/sql-tests/testcases/tpcds/q33.sql]: squash?

Comment (findepi) [reply presto-product-tests/src/main/resources/sql-tests/testcases/tpcds/q33.sql]: This one is in the product tests.

**Comment by findepi** [comment]
> Make sure they are passing

I did :)

**Pull request merged**